### ISSUE TITLE
Regularly "processing items due to sync_list being used", but I don't have a sync_list

### ISSUE THREAD

**Issue by malord** [abraunegg/onedrive#755]
While investigating why a file hadn't turned up on my Linux box (it did eventually), I found that I regularly get many of these in my log:

    Processing 229 OneDrive items to ensure consistent state due to sync_list being used

There were 71 of these the last time it did it. The thing is, I'm not using sync_list. I've tried:

    onedrive --resync --synchronize

But it has not improved the situation.

I'm using 2.3.12 on Ubuntu 18.04 and DMD64 v2.089.0.

**Comment by malord** [comment]
Should also add, if I make a change while the client is doing this, I get:

    Error Message: HTTP request returned status code 500 (Internal Server Error)
    Error Reason:  Failed to get a coherent view of changes. Try again later.

and it then does a lengthy resync.

**Comment by abraunegg** [comment]
@malord 
Regarding this error:
```
Processing 229 OneDrive items to ensure consistent state due to sync_list being used
```
Please can you test this PR for resolution of this issue:

1. Stop any running onedrive systemd service

```
git clone https://github.com/abraunegg/onedrive.git
cd onedrive
git fetch origin pull/745/head:pr745
git checkout pr745
./configure --enable-notifications; make clean; make;
```
2. Install the PR
3. Restart onedrive systemd process

Regarding:
```
Error Message: HTTP request returned status code 500 (Internal Server Error)
Error Reason:  Failed to get a coherent view of changes. Try again later.
```
This is normal.


Once done, please advise.

**Comment by abraunegg** [comment]
@malord 
Have you had a chance to test the PR yet?

**Comment by malord** [comment]
Been running it since last night and not yet seen the sync_list message in the logs. Will keep observing it as I use OneDrive today. Is there anything else I should look out for in this PR?

**Comment by abraunegg** [comment]
@malord 
No - that is the main fix for that PR - fixing the inconsistent messaging

**Comment by abraunegg** [comment]
Marking bug as fixed, closing as PR #745 merged into master
